### PR TITLE
Every 5 seconds, drop charts that haven't received data

### PR DESCRIPTION
**Pull request by caitlin-hennessy**

### CODE REVIEW

File: webapp/src/serial.tsx [hunk 1]
```diff
@@ -147,6 +149,18 @@ export class Editor extends srceditor.Editor {
         }
     }
 
+    dropStaleCharts() {
+        let now = Util.now()
+        for (let i = 0; i < this.charts.length; ++i) {
```
Comment (pelikhan) [inline webapp/src/serial.tsx]: use ``this.charts.forEach(chart => ...)``

File: webapp/src/serial.tsx [hunk 2]
```diff
@@ -54,6 +55,7 @@ export class Editor extends srceditor.Editor {
 
     constructor(public parent: pxt.editor.IProjectView) {
         super(parent)
+        setInterval(this.dropStaleCharts.bind(this), 5000)
```
Comment (pelikhan) [inline webapp/src/serial.tsx]: should only be started be active while editor is active

Comment (caitlin-hennessy) [reply webapp/src/serial.tsx]: Fixed

File: webapp/src/serial.tsx (hunk 1 above)
Comment (caitlin-hennessy) [reply webapp/src/serial.tsx]: Fixed

**Pull request merged**